src/app/pages/clients/clients.component.html

```diff
@@ -0,0 +1,61 @@
+<div>
+  <div class="row">
+    <div class="col-md-8">
+      <div class="row">
+        <div class="col-md-6" *ngFor="let client of clients">
+          <div class="card listview">
+            <div class="row no-gutters">
+              <div class="col">
+                <div class="card-block px-2">
+                  <h4>
+                    <b class="card-title"> {{ client.client_name }}</b>
+                  </h4>
+
+                  <ul class="list-unstyled fa-ul">
+                    <li>
+                      <i class="fa fa-street-view"></i>
+                      <span
+                        >{{ client.address.Street }}, {{ client.address.City }},
+                        {{ client.address.State }}
+                      </span>
+                    </li>
+                    <li>
+                      <i class="fa fa-phone"></i>
+                      <span>{{ client.phone }}</span>
+                    </li>
+                    <li>
+                      <i class="fa fa-envelope"></i
+                      ><span>{{ client.email }}</span>
+                    </li>
+                  </ul>
+                </div>
+              </div>
+            </div>
+            <div class="card-text">
+              <p>
+                <b class="client-reviews">Reviews</b>
+
+                <span class="reviews">
+                  <a href="/client/:clientId/review" class="card-link"
```

Comment:
Nice work, but this links seam to be static. They all point to `/client/:clientId/review`. What do you think of making it replace the `:clientId` with an actual `id`

Reply:
@Tevinthuku I think there's another PR up for that. #42

Reply:
@Rhytah maybe you can just have a `#` for the links here to avoid the confusion.

Reply:
@wangonya , tht one is for reviewing a client, but don't you think this PR should at least provide the link to the page, even though it doesn't exist at this point. It could make it easier since @Rhytah won't have to come back again to edit the links

Reply:
@Tevinthuku that link is static for now because as @wangonya pointed out that's outside the scope of my story as it is being worked on by another developer. After integrating the functionality of the other feature is when they will become dynamic.

Reply:
@Tevinthuku, Thank you for pointing it out. @Rhytah & @wangonya The PR for adding a review assumes that the client ID will be dynamic and will get it from the route. So, it would be better to provide a dynamic client ID instead of coming back later and edit the links.